overview.qmd

```diff
@@ -70,6 +70,12 @@ and release version of base R from the same day that the R-multiverse snapshot w
 R-multiverse is powered by [R-universe](https://r-universe.dev/), [GitHub Actions](https://github.com/features/actions),
 and the public source code repositories in the [R-multiverse GitHub organization](https://github.com/r-multiverse).
 
+## How you can help
+
+* Share R-multiverse with your colleagues.
```

Comment:
```suggestion
* Spread the word: Share R-multiverse with your colleagues.
```

Reply:
added in a82f4ffae3f9ded3652454dbaabc471fc269d954 (sorry, didn't see the "commit suggestion" button until just now)

Reply:
@wlandau an even better button is the "Add suggestion to batch" one but it's only available from the Files tab of a PR, not from the Discussion tab :zany_face: